localstack/dns/server.py

```diff
@@ -793,15 +835,40 @@ def start_server(upstream_dns: str, host: str, port: int = config.DNS_PORT):
         dns_server.shutdown()
         return
     DNS_SERVER = dns_server
+    LOG.debug("DNS server startup finished!")
```

Comment:
nit: I feel the exclamation mark is not inline with our current messages
```suggestion
    LOG.debug("DNS server startup finished")
```